recipe/meta.yaml

```diff
@@ -110,14 +110,14 @@ outputs:
     build:
       string: h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}_{{ build_ext }}
       run_exports:
-        - {{ pin_subpackage("libarrow", max_pin="x") }}
-        - {{ pin_subpackage("libarrow-acero", max_pin="x") }}
-        - {{ pin_subpackage("libarrow-dataset", max_pin="x") }}
-        - {{ pin_subpackage("libarrow-flight", max_pin="x") }}
-        - {{ pin_subpackage("libarrow-flight-sql", max_pin="x") }}
-        - {{ pin_subpackage("libarrow-gandiva", max_pin="x") }}
-        - {{ pin_subpackage("libarrow-substrait", max_pin="x") }}
-        - {{ pin_subpackage("libparquet", max_pin="x") }}
+        - {{ pin_subpackage("libarrow", max_pin="x.x") }}
```

Comment:
> Am noticing [in `pyarrow`](https://github.com/conda-forge/pyarrow-feedstock/blob/64ce0731d0a5e59c0e293c80b5ae7202d30ee021/recipe/meta.yaml#L89-L92), we are doing this

This is necessary because pyarrow is built on a different feedstock, and so cannot use `pin_subpackage`. 

> Do we want a similarly tight pin here or should `pyarrow` relax its pin?

Neither.